Makefile

```diff
@@ -17,4 +17,7 @@ clean-game:
 emake:
 	$(MAKE) -C CommandLine/emake/
 
+test-runner:
```

Comment:
The test runner expects `emake` to be built, so the `test-runner` target should depend on the `emake` one.

Reply:
Good point. I've added a `.FORCE` rule to cause the actual make rule to be run, and I've added the dependency you mentioned.